test/functions/test_base.rb

```diff
@@ -229,6 +229,25 @@ def test_normalize_space
       assert_equal( [REXML::Comment.new("COMMENT A")], m )
     end
 
+    def test_normalize_space2
```

Comment:
Could you use more meaningful test name something like:

```suggestion
    def test_normalize_space_strings
```